### ISSUE TITLE
[1.18.2] Colors do not display correctly

### ISSUE THREAD

**Issue by SylvanusPC** [ichttt/McPaint#18]
Hello,

I am currently using MC Paint version 1.8.0 on Minecraft 1.18.2 with Forge 40.1.30. I am having an issue on a multiplayer server where painting works fine, but for whatever reason all colors are switched to black when viewing. The colors show up fine in the editor, but not outside it. Attached are screenshots of the issue occurring. The only conflict I can think of might be with the mod True Darkness, which disables Minecraft's minimum light level, but I do not know why it might be conflicting. Any help would be appreciated.

P.S. - Fantastic mod! Just what I was looking for

![image](https://user-images.githubusercontent.com/30609492/176517646-4cc87188-4ff9-4fd8-b6f4-cadb698792b0.png)

![image](https://user-images.githubusercontent.com/30609492/176517709-e29d2f6b-3098-4865-847e-7077176cd691.png)

**Comment by SylvanusPC** [comment]
I have an update! When having night vision effect, the colors display fine. Without it, they are black. This also makes me think it's a conflict with True Darkness. Is there a minimum light level needed for display of these colors?

**Comment by ichttt** [comment]
I'll take a look at it when I have some time. Do you have any other mods installed?

**Comment by SylvanusPC** [comment]
Yes, other mods are installed. Testing without other mods works fine, however to test compatibility with every mod in this pack would mean dissecting it one by one, something which would take a long amount of time and I believe to be unproductive. I think the issue seems to be compatibility with the mod True Darkness: True Darkness modifies Minecraft to remove the minimum light level, so dark areas are pitch-black. Having "night vision" effect allows colors to display normally, but without it they show up as black, even when in a lit area. I tried several variations of light level, including the maximum, but it will not make the colors appear without night vision. Hope this helps.

**Comment by gio3k** [comment]
i'm getting the same issue; it seems to work fine on Mac while pictures have dark colours on Windows running the same modpack
